backend/apps/github/common.py

```diff
@@ -46,7 +46,7 @@ def sync_repository(gh_repository, organization=None, user=None):
     )
 
     # GitHub repository issues.
-    if not repository.is_archived and repository.project:
+    if not repository.is_archived and repository.project and repository.sync_issues:
```

Comment:
You need to add the project level flag check too

Reply:
making it this way?
-----------------------------------------------------
    if not repository.is_archived and repository.project and repository.sync_issues and repository.project.sync_issues:

Reply:
I made an edit instead:
    if not repository.is_archived and repository.project and repository.sync_issues and repository.project.is_indexable:

also in project.py I modified the is_indexable function in Project model to:
def is_indexable(self):
        """Projects to index."""
        return self.is_active and self.has_active_repositories and self.sync_issues

Reply:
which way should I consider?

Reply:
At this step let's stick with the first option that involves `sync_issues` flags only (don't touch `is_indexable`)